fire/api/model/observationer.py

```diff
@@ -507,3 +513,198 @@ class Nulobservation(Observation):
     """
 
     __mapper_args__ = {"polymorphic_identity": ObservationstypeID.nulobservation}
+
+
+class ObservationsLængde(Observation):
+    """
+    Observationslængden af en GNSS-måling.
+
+    Bemærk at observationstidspunktet angiver starten af observationsperioden.
+    Ved hjælp af observationvarigheden kan både sluttidspunktet og
+    "middelobservationstidspunktet" beregnes.
```

Comment:
Er `midelopbservationstidspunkt = obervationsstart + varighed / 2` ?

Reply:
Noget i den stil - detaljerne kan utvivlsomt findes i `fire.io.bernese` (jeg har ikke forholdt mig yderligere til det i denne omgang).

Reply:
Ok. Tak for referencen.